src/js/modules/infragistics.ui.editors.js

```diff
@@ -8081,6 +8095,44 @@
 
 			this._super();
 		},
+		_validateSpinSettings: function() {
+			var delta = this.options.spinDelta, deltaRanges, index;
+
+			if (typeof delta === "number") {
+				if (delta < 0) {
+					throw new Error(this._getLocaleValue("spinDeltaCouldntBeNegative"));
+				} else {
+					this.options.spinDelta = parseInt(delta, 10);
+				}
+			} else if (typeof delta === "object") {
+				deltaRanges = [ [ "years", 10 ], [ "months", 12 ], [ "days", 28 ],
+					[ "hours", this._dateIndices.hh24 ? 24 : 12 ], [ "minutes", 60 ],
+					[ "seconds", 60 ], [ "milliseconds", 1000 ] ];
+
+				for (index = 0; index < deltaRanges.length; index++) {
+					this._validateSpinSettingsPeriod.apply(this, deltaRanges[ index ]);
+				}
+			} else {
+				throw new Error(this._getLocaleValue("spinDeltaIsOfTypeNumberOrObject"));
+			}
+		},
+		_validateSpinSettingsPeriod: function(name, maxRange) {
+			var delta = this.options.spinDelta[ name ];
+
+			if (delta === undefined) {

```

Comment:
This can be skipped entirely by not validating undefined values at all (skipping other checks) and simply returning a default 1 in `_getDelta`?